config.go

```diff
@@ -0,0 +1,33 @@
+package rossby
+
+import (
+	"github.com/dgraph-io/badger"
+	"github.com/kelseyhightower/envconfig"
+)
+
+// The prefix used to load environment variables with envconfig
+const envvarPrefix = "rossby"
+
+// Config stores the optional settings that modify the behavior of rossby at runtime.
+// A config can be created either through default values specified on the struct tags,
+// or fetched from environment variables prefixed with ROSSBY_.
+//
+// TODO: change the database default path after development
+// TODO: allow configuration from cli context or from YAML file
+type Config struct {
+	Address  string `default:":1205"`                   // the host and port to bind the server to ($ROSSBY_ADDRESS)
+	Database string `default:"fixtures/db"`             // the path to the directory of the badger database ($ROSSBY_DATABASE)
+	LogLevel int    `default:"3" envconfig:"log_level"` // verbosity of logging, lower is more verbose ($ROSSBY_LOG_LEVEL)
+}
+
+// Load the configuration from the environment and set default values.
+func (c *Config) Load() (err error) {
+	return envconfig.Process(envvarPrefix, c)
+}
```

Comment:
Ideally, we have a configuration system that goes as follows:

- load defaults from struct tags
- load configs from configuration files
- load from environment
- load from cli context (e.g. flags parsed from the command line) 

I've looked in a few places for this and can't find anything that meets all of my requirements. Because we're probably going to be doing this in containers, I've opted for a simple configuration based on the environment and defaults.